source/Octopus.Client.Tests/Integration/OctopusClient/OctopusAsyncTestClient.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.IO;
+using System.Net;
+using System.Reflection;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using Octopus.Client.Model;
+using Octopus.Client.Serialization;
+
+namespace Octopus.Client.Tests.Integration.OctopusClient
+{
+    public class OctopusAsyncTestClient : OctopusAsyncClient
```

Comment:
I had a few issues where it wasn't deserialising to the correct type, so I wanted to make sure it was testing the whole thing. The cycle time from make the code change, run the tests, run the build, pull the nupkg into the API tests, update the API tests, run the API tests, find the deserialisation error, repeat... was pretty high, so wanted to make sure it was tested locally.